### PR TITLE
Natively display floorplan on rviz

### PR DESCRIPTION
**Pull request by youliangtan**
This feature will help in the visualization of the existing building floorplan on Rviz. Basically, these floorplans are those being annotated in  `TrafficEditor`.

Floorplan will be generated and published within `rmf_scedulle_visualizer` to topic `/floorplan`, using classic ros occupancy grid msg format. Briefly speaking, the floorplan data here is decoded from `buildingmap` msg, provided by the `building_map_server`.

### CODE REVIEW

File: .github/workflows/build_eloquent.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
         echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable bionic main" > /etc/apt/sources.list.d/gazebo-stable.list
         wget https://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
         sudo apt update
-        sudo apt install python3-shapely python3-yaml python3-requests \
+        sudo apt install python3-shapely python3-yaml python3-requests libopencv-dev \
```
Comment (Yadunund) [inline .github/workflows/build_eloquent.yaml]: Looks like libopencv-dev can be installed via `rosdep` if we add it to the `package.xml` file?

File: rmf_schedule_visualizer/src/main_rviz.cpp [hunk 2]
```diff
@@ -712,6 +725,47 @@ class RvizNode : public rclcpp::Node
...
+  void publish_floorplan()
+  {
+    if (_level.images.size() == 0)
+      return;
+
+    auto floorplan_img = _level.images[0]; // only use the first img
+    RCLCPP_INFO(this->get_logger(),
+      "Loading floorplan Image: " + floorplan_img.name +
+      floorplan_img.encoding);
+    std::cout<<""<<std::endl; // flush RCLCPP_INFO printout, fixed in foxy
```
Comment (Yadunund) [inline rmf_schedule_visualizer/src/main_rviz.cpp]: Do we still need this line?

**Review by Yadunund — CHANGES_REQUESTED** [review]
This is awesome! Thanks for visualizing floor plans via occupancy grids. Left a couple of comments for your consideration. Could you also help to update the changelog for this package?

File: .github/workflows/build_eloquent.yaml (hunk 1 above)
Comment (youliangtan) [reply .github/workflows/build_eloquent.yaml]: Gotcha, will try test it out the CI

**Comment by Yadunund** [comment]
CI is failing for foxy as the `foxy` branch in `rmf_core` has been merged into master and deleted. So can delete the `-b foxy` from https://github.com/osrf/rmf_schedule_visualizer/blob/d1872cb1d88e72ba29abfeb96cd3fe25833449d6/.github/workflows/build_foxy.yaml#L25 

Can also delete the eloquent build file since we are no longer supporting eloquent. But that can be a separate PR

**Review by Yadunund — APPROVED** [review]
CI passed for foxy

**Comment by youliangtan** [comment]
At the same time, removed `eloquent` build and update docs to `foxy`

**Pull request merged**